Documentation/compatibility/winforms-hdpi-changes-48.md

```diff
@@ -0,0 +1,84 @@
+## High DPI (HDPI) improvements in Windows Forms controls for .NET 4.8
+
+### Scope
+Major
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Forms Framework is continuing to improve the scaling and rendering of its controls to better support Windows Forms customers on high resolution devices. These include the following changes:
+- Changes to improve the scaling of PropertyGrid when the DPI of the application changed.
```

Comment:
Can we add an explanation of "when the DPI of the application changed" like we change monitor setting while the application is running or we drag an application to another monitor with a different DPI setting